src/sagemaker/jumpstart/types.py

```diff
@@ -895,6 +909,15 @@ def from_json(self, json_obj: Dict[str, Any]) -> None:
                 else None
             )
 
+    def from_hub_content_doc(self, hub_content_doc: Dict[str, Any]) -> None:
+        """Sets fields in object based on values in HubContentDocument
+
+        Args:
+            hub_content_doc (Dict[str, any]): parsed HubContentDocument returned
+                from SageMaker:DescribeHubContent
+        """
+        # TODO: Implement
```

Comment:
don't you need a `pass` here?

Reply:
Nope, pylint throws an error if I add one